satpy/etc/readers/viirs_l2_lsr.yaml

```diff
@@ -0,0 +1,187 @@
+reader:
+    description: VIIRS Land Surface Reflectance Reader
+    name: viirs_l2_lsr
+    reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+    sensors: [viirs]
+
+file_types:
+    lsr_netcdf:
+        file_reader: !!python/name:satpy.readers.viirs_l2_lsr.VIIRSLandSurfaceReflectanceFileHandler
+        variable_prefix: ""
+        file_patterns: 
+            - 'SurfRefl_v{version:1d}r{revision:1d}_{platform_shortname:3s}_s{start_time:%Y%m%d%H%M%S%f}_e{end_time:%Y%m%d%H%M%S%f}_c{creation_time:%Y%m%d%H%M%S%f}.nc'
+
+navigations:
+    vgeoi:
+        description: VIIRS L1B I-band Navigation
+        file_type: vgeoi
+        latitude_key: Latitude_at_375m_resolution
+        longitude_key: Longitude_at_375m_resolution
+        nadir_resolution: [371]
+        rows_per_scan: 32
+    vgeom:
+        description: VIIRS L1B M-band Navigation
+        file_type: vgeom
+        latitude_key: Latitude_at_750m_resolution
+        longitude_key: Longitude_at_750m_resolution
+        nadir_resolution: [742]
+        rows_per_scan: 16
+
+datasets:
+    i1:
+        name: i1
+        standard_name: i1
+        file_type: [lsr_netcdf]
+        file_key: "375m Surface Reflectance Band I1"
+        coordinates: [i_lon, i_lat]
+        units: '1'
+        _FillValue: -9999
+    srf_i1:
+        name: srf_i1
+        standard_name: surface_reflectance
```

Comment:
I think the technical CF standard name is `surface_bidirectional_reflectance`. @kathys does that sound accurate for these data files?

Reply:
Yes, that is correct.